src/components/JsxParser.js

```diff
@@ -61,7 +64,7 @@ export default class JsxParser extends React.Component {
     const components = this.props.components.reduce(
       (map, type) => ({
         ...map,
-        [type.prototype.constructor.name.toUpperCase()]: type,
```

Comment:
I know it's not part of this PR, but using reduce() with an Array here relies on components having a `.name` property, which will be mangled in any production setup (eg: uglify).  Some people define a displayName on the constructor, which would be consistently reliable.  Or, it might be worth supporting either an Array (current) or object (bypassing reduce()).  

Here's an example of where the object is very useful:

```js
<JsxParser components={{
  // these classes wouldn't likely be named "A" or "IMG"
  a: Link,
  img: LazyImage
}} />
```